lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb

```diff
@@ -0,0 +1,78 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionAddTestersAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        if blank?(params[:emails]) && blank?(params[:file])
+          UI.user_error!("Must specify `emails` or `file`.")
+        end
+
+        emails = get_value_from_value_or_file(params[:emails], params[:file]).split(',')
+
+        if emails.count > 1000
+          UI.user_error!("A maximum of 1000 testers can be added at a time.")
+        end
+
+        fad_api_client.add_testers(params[:project], emails)
+
+        UI.success("✅ Tester(s) successfully added.")
```

Comment:
Even though it's not as meaningful as in the delete case, but how about getting the count in there just for reference, e.g. `"#{emails.count} tester(s) successfully added."`?

Reply:
Thought about this, but we technically wouldn't be able to know exactly how many testers were actually created. The response contains all the testers from the request even if they exist already, so there could be testers in there that weren't actually created.

Reply:
Got it. Could we add that note as a comment to make sure this message stays vague?

Reply:
Considered this but in the response we include testers regardless of whether or not they were created or already existed so it technically wouldn't be an accurate count of then number of testers that were actually created.

Reply:
@rebehe That wouldn't be accurate either, since the project could already have other testers that were not included in the request.

@tagboola Your comment LGTM, thanks.